core/src/test/java/kafka/server/SharePartitionManagerTest.java

```diff
@@ -1128,4 +1135,53 @@ public void testShareFetchContextResponseSize() {
         assertEquals(Errors.NONE, resp8.error());
         assertEquals(4 + resp8.data().size(objectSerializationCache, version), respSize8);
     }
+
+    @Test
+    public void testAcknowledge() {
```

Comment:
I am sure there would be tons of unit tests for `Acknowledge`, are we planning to add further in next PR?